homework/index.js

```diff
@@ -30,18 +30,96 @@
     return elem;
   }
 
+  function drawRepository(data, value, container) {
+    // Clear the details of the selected option
+    container.innerHTML = '';
+    drawRepositoryInfo(data, value, container);
+    drawRepositoryContributors(data, value, container);
+  }
+
+  function drawRepositoryInfo(data, value, container) {
+    const leftDiv = createAndAppend('div', container, { class: 'left-div' });
+    const repoTable = createAndAppend('table', leftDiv);
+    const repoTbody = createAndAppend('tbody', repoTable);
+
+    const tr1 = createAndAppend('tr', repoTbody);
```

Comment:
This is only a styling thing, I would prefere to have the names instead of `tr1`, `tr2` ...etc, names like `repoNameRow`, `repoDescriptionRow` ...etc
it looks more clear for me, what do you say 🙂